First webpage/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Amy Brown: Medieval Literatures, Modern Cultures</title>
+    <link rel="stylesheet" type="text/css" href=style.css>
+  </head>
+  <body>
+
+    <img src="header-image-crop.jpg" class="headerimage" width="1180" alt="a red cushioned chair holding books on Old and Middle English literature and medieval sexuality">
+  
+  
+    <div class="invisiblebox">
+    <div class="mainbox">
+      <h1>Amy Brown: Medieval Literatures, Modern Cultures</h1>
+      <h2>Researcher in Medieval Literatures and Contemporary Reception</h2>
+    
+      <div class="row">
+        <div class="column">      
+          <p>I'm an early career researcher with a wide range of interests and teaching experience. I've worked on both medieval English and medieval French, and from there my interests in translation and adaptation have joined my interests in gender and sexuality to bring me into the field of contemporary adaptation and reception. </p>
+        </div>
+        <div class="columngap"></div>
+        <div class="column">
+          <p>Current post: Postdoctoral Researcher on the project Canonicity, Obscenity, and the Making of Modern Chaucer (COMMode), led by <a href=http://www.marycflannery.com/>Prof. Dr. Mary Flannery</a>
+            <br>University of Bern (<a href=https://www.ens.unibe.ch/about_us/staff/dr_brown_amy/index_eng.html>Department profile</a> | Swiss National Science Foundation</p>
+          <p>PhD., University of Geneva (2018)</p>
+          <p>Email: <a href=mailto:amybrown@amybrownresearch.net>amy.brown@amybrownresearch.net</a>
+          <br>Twitter: <a href=https://twitter.com/amisamileandme>@amisamileandme</a></p>         
+        </div>
+      </div>
+  </div>
+
+  <div class="flexboxfooters">
+  <div class="flexchild"><p><li>Publications</li><br>
+<li>Academic CV</li><br>
+<li>Teaching History</li></p></div> 
+<div class="flexchild">
+   <p>LINK: NEWS / UPDATES</p>
+ </div>
+ </div>
```

Comment:
For this "flexboxfooters" div it would be perfect to use a <footer> HTML element instead.

Also, this whole section would really benefit from thinking about indentation so it is easier to understand what we are looking at.